src/components/Camera/index.js

```diff
@@ -1,3 +1,5 @@
+// @flow
```

Comment:
are these necessary? I don't like libraries that use magic comments (ahem [ruby](https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-string-literal-true-do))

Reply:
necessary. you can remove it if you turn flow for every file I think

Reply:
the `jsx` pgrama mark is necessary though, even though it should be taking it from the babel config.